src/components/Course/Course.vue

```diff
@@ -262,27 +291,31 @@ export default defineComponent({
 <style scoped lang="scss">
 @import '@/assets/scss/_variables.scss';
 
+.course-container {
+  position: relative;
+  padding-bottom: 20px;
+}
+
 .course {
+  box-shadow: 0px 0px 10px 4px rgba(0, 0, 0, 0.055);
+  position: relative;
+  z-index: 1;
```

Comment:
This is the `z-index` I was referencing previously — seems to mess with the course menu dropdown precedence.